databases/sqlcipher/Portfile

```diff
@@ -0,0 +1,114 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+
+name                sqlcipher
+# don't forget to update the checksums for sqlite3-tools when updating sqlite3
+revision            0
+categories          databases
+platforms           darwin
+license             BSD
+
+maintainers         {gmail.com:ybzhao1989 @z7z8th} openmaintainer
+
+description         an security extesion to the embedded SQL database engine
+
+long_description    SQLCipher extends the SQLite database library to add \
+                    security enhancements that make it more suitable for \
+                    encrypted local data storage like: \
+                     \* on-the-fly encryption\
+                     \* tamper detection\
+                     \* memory sanitization\
+                     \* strong key derivation\
+                    \
+                    SQLite3 is an SQL database engine in a C library. \
+                    Programs that link the SQLite3 library can have SQL \
+                    database access without running a separate RDBMS \
+                    process. The distribution comes with a standalone \
+                    command-line access program (sqlite3) that can be used \
+                    to administer an SQLite3 database and which serves as \
+                    an example of how to use the SQLite3 library.
+
+homepage            https://www.zetetic.net/sqlcipher/
+github.setup        sqlcipher sqlcipher 4.4.2 v
+
```

Comment:
please add `github.tarball_from archive`  or `release` and that will require to update the `checksums`

Reply:
added `github.tarball_from archive` since no release in sqlcipher repo.
also added openssl11 as dependency to fix the workflow checks.